relay-server/src/services/processor.rs

```diff
@@ -1554,6 +1555,21 @@ impl EnvelopeProcessorService {
             unreal::expand(managed_envelope, &self.inner.config)?;
         });
 
+        // TODO: Check if this is the correct way of doing this. Or if we should go via
+        // should_filter.
+        if managed_envelope
+            .envelope()
+            .required_features()
+            .contains(&Feature::PlaystationEndpoint)
+            && !project_info.has_feature(Feature::PlaystationEndpoint)
```

Comment:
Should be hidden in `playstation::expand`

Reply:
Looking at the other spots where we do the filtering it happens on the top level, but can move it into `expand`.